recipes/kernel-tuner/meta.yaml

```
@@ -0,0 +1,76 @@
+{% set name = "kernel-tuner" %}
+{% set version = "1.0.0b6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/kernel_tuner/kernel_tuner-{{ version }}.tar.gz
+  sha256: 8ec7cac41f1eec84e54bf2452008c905fad90279949e477355fb9eda054b1beb
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: True  # [win or py>311]
```

This looks like it can be noarch
```suggestion
  noarch: python
```

---

I tried that initially but the tests fail on Windows (see [580d294](https://github.com/conda-forge/staged-recipes/pull/24420/commits/580d294703d49ccf4354ed61e7983903152f2e47) and [here](https://github.com/conda-forge/staged-recipes/pull/24420/checks?check_run_id=18489577765)), so after discussing with the authors of the package in https://github.com/KernelTuner/kernel_tuner/issues/128#issuecomment-1802186318, I disabled the windows build to avoid disappointing Windows users by allowing them to install something that doesn't work. Is this the right way to do it? Or can we make the package `noarch: python` but still skip the Windows build?

---

You can make it `noarch: python` but add a `run` dependency on `__unix`. That would avoid building the package for every Python/OS combination.

---

I changed this as suggested in [52be0d1](https://github.com/conda-forge/staged-recipes/pull/24420/commits/52be0d105aac6eb456457d669e92b78a04f3e4fb), but now the windows build is failing again. Did I miss something?